channels/serializers.py

```diff
@@ -229,6 +231,14 @@ def get_upvoted(self, instance):
         """Is a comment upvoted?"""
         return instance.likes is True
 
+    def get_profile_image(self, instance):
+        """Find the Profile for the comment author"""
+        profile = self.context['request'].user.profile
+        if (profile and profile.image_small):
```

Comment:
The parentheses around the if expression are unnecessary in Python. Also `profile` should exist for all users since it is created by a signal when the user is created